README.md

```diff
@@ -20,28 +20,28 @@ Below, we illustrate a minimal sample configuration with the international SQLit
 
 > warning: This beta version  of the Launcher 2.0 is for test purpose only and should be used with our German sandbox.
 
-Download the latest release from GitHub.
+Download the latest release from GitHub. We always recommend using the latest release to benefits from the newest improvements.
 Unzip the downloaded release.
 
-### Supported Packages in the Alpha
+### <a name="supported-packages-in-the-alpha"></a>Supported Packages in the Alpha
```

Comment:
```suggestion
### Supported Packages in the Alpha
```

I don't think this is needed as the links to headings are automatically generated by github 🤔 If this has another reason ignore me ^^

Reply:
Sorry I added that because when i tested in Visual studio code the links created in the guide did not work when using the heading